cves/CVE-2014-0095.yml

```diff
@@ -130,8 +137,8 @@ subsystem:
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
     Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer:
-  name:
+  answer: The AJP request processor
+  name: ajp
```

Comment:
```suggestion
  name: AJP
```

Reply:
Based on the examples given I assume this is supposed to be lowercase.